src/gcp/location.ts

```diff
@@ -0,0 +1,112 @@
+// A mapping from geographical region to subdomain, usefull for Container Registry
+const SUBDOMAIN_MAPPING: Record<string, string> = {
```

Comment:
If this is strictly GCR related it can possibly be in `docker.ts` and be something like `GCR_SUBDOMAIN_MAPPING` (I know we've made you move this already)

Reply:
Moved